src/ngx_lua_resty_lmdb_module.c

```diff
@@ -16,12 +16,12 @@ static ngx_int_t ngx_lua_resty_lmdb_init_worker(ngx_cycle_t *cycle);
 static void ngx_lua_resty_lmdb_exit_worker(ngx_cycle_t *cycle);
 
 
-static int get_digest_key(ngx_str_t *passwd, MDB_val *key);
-static int lmdb_encrypt_func(const MDB_val *src, MDB_val *dst,
-                        const MDB_val *key, int encdec);
+static int ngx_lua_resty_lmdb_digest_key(ngx_str_t *passwd, MDB_val *key);
+static int ngx_lua_resty_lmdb_cipher(const MDB_val *src, MDB_val *dst,
+                                     const MDB_val *key, int encdec);
 
 
-static const EVP_CIPHER *cipher;
+static const EVP_CIPHER *g_cipher;
```

Comment:
I think this should just be moved into `ngx_lua_resty_lmdb_conf_t`, using global variables is not so great looking.

Reply:
Could we do it in another PR? This PR is just for naming style.

Reply:
We can not move `cipher` into `ngx_lua_resty_lmdb_conf_t`, because the encrypt function `ngx_lua_resty_lmdb_cipher` has a fixed signature, we could only pass `cipher` with global var form.

Reply:
done.